extension.webpack.config.cjs

```diff
@@ -128,4 +128,28 @@ const report = {
   ...commonConfig,
 };
 
-module.exports = [root, devTools, popup, report];
+const dashboard = {
+  entry: {
+    index: './src/view/devtools/dashboard.tsx',
+  },
+  output: {
+    path: path.resolve(__dirname, './dist/extension/devtools'),
+    filename: 'devtoolsDashboard.js',
+  },
+  plugins: [
+    new WebpackBar({
+      name: 'Dashboard',
+      color: '#FF9B71',
+    }),
+    new HtmlWebpackPlugin({
+      title: 'Report',
+      template: './src/view/devtools/dashboard.html',
+      filename: 'dashboard.html',
+      inject: 'body',
+    }),
+    new HtmlInlineScriptPlugin(),
+  ],
+  ...commonConfig,
+};
```

Comment:
This would be confusing, let's rename this to something like dashboard report or something.